Failing test on the backend for `test_get_teros_obj_fraction_scales_to_percent`

We should not have failing tests merged into main.

See the following link for the full logs:
https://github.com/jlab-sensing/ENTS-backend/actions/runs/22733588009/job/65928917596

Relevant snipped
```
init_database = <FlaskClient <Flask 'api'>>

    def test_get_teros_obj_fraction_scales_to_percent(init_database):
        """
        GIVEN adjusted VWC stored as a fraction
        WHEN TEROS data is retrieved
        THEN VWC is returned as a percentage
        """
        ts = 1705176162
        formated_ts = datetime.fromtimestamp(ts)
        cell = Cell("cell_9", "", 1, 1, False, None)
        cell.save()
        TEROSData.add_teros_data("cell_9", formated_ts, 0.42, 2, 3, 4, 5)
    
        teros_data_obj = TEROSData.get_teros_data_obj(
            cell.id, "none", formated_ts, formated_ts, False
        )
    
        assert 42 in teros_data_obj["vwc"]
>       assert teros_data_obj["vwc_unit"] == "%"
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
E       KeyError: 'vwc_unit'

```

--- a/backend/api/models/teros_data.py
+++ b/backend/api/models/teros_data.py
@@ -108,7 +108,14 @@ class TEROSData(db.Model):
         if end_time is None:
             end_time = datetime.now()
 
-        data = {"timestamp": [], "vwc": [], "temp": [], "ec": [], "raw_vwc": [], "vwc_unit": "%"}
+        data = {
+            "timestamp": [],
+            "vwc": [],
+            "temp": [],
+            "ec": [],
+            "raw_vwc": [],
+            "vwc_unit": "%",
+        }
 
         stmt = None
 

